ts/src/pro/coinbase.ts

```diff
@@ -724,6 +724,28 @@ export default class coinbase extends coinbaseRest {
         return message;
     }
 
+    handleHeartbeats (client, message) {
+        // although the subscription takes a product_ids parameter (i.e. symbol),
+        // there is no (clear) way of mapping the message back to the symbol.
+        //
+        //     {
+        //         "channel": "heartbeats",
+        //         "client_id": "",
+        //         "timestamp": "2023-06-23T20:31:26.122969572Z",
+        //         "sequence_num": 0,
+        //         "events": [
+        //           {
+        //               "current_time": "2023-06-23 20:31:56.121961769 +0000 UTC m=+91717.525857105",
+        //               "heartbeat_counter": "3049"
+        //           }
+        //         ]
+        //     }
+        //
+        // treat this as a pong, to avoid ping-timeout-error
+        client.onPong ();
```

Comment:
My thought was: it's a sign of being alive; _like_ a pong. 
The thing is: If I ommit the `onPong`, I get a lot of `timed out due to a ping-pong keepalive missing on time` messages. That's pulling the plug ourselves, although we know that the connection is alive.

Reply:
@CH-EdwinB hum in that case I think it might be ok to call onPong I think

Reply:
I have removed the call to `onPong`, because I actually agree that it doesn't feel right to call `onPong`.

First of all, because it's not an actual pong.

But more important:
Coinbase server has not responded to a ping request within the given timeframe.
I have tested it with keepAlive = 60000 and maxPingPongMisses = 2.0. 
I still get the timeout exception, which means that coinbase is more than 2 minutes too late with the pong, even though it sends (other) messages on the connection. This is a coinbase problem, not a ccxt problem. 

I will inform coinbase that they need to fix this asap.